setup_wordpress.sh

```diff
@@ -4,3 +4,68 @@ set -e
 
 # Your code goes below ...
 echo 'This script should install and setup Wordpress'
+
+echo '---------- Update & Upgrade APT ----------'
+sudo apt-get update -y
+sudo apt-get upgrade -y
+
+echo '---------- Install Nginx Package and configure Firewall ----------'
+sudo apt-get install nginx -y
+sudo ufw allow 'Nginx Full'
+
+echo '---------- Install MYSQL Packages ----------'
+sudo apt-get install mysql-server -y
+
+echo '---------- Create WordPress Database & User ----------'
+mysql_pw='YourPassword123!'
+sudo debconf-set-selections <<< "mysql-server mysql-server/root_password password ${mysql_pw}"
```

Comment:
what does this line?

Reply:
debconf-set-selections lets you install mysql unattended, mysql prompt asks for password then we set the password.